common/libs/signing.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import EthTx from 'ethereumjs-tx';
+import { sha3, ecsign } from 'ethereumjs-util';
+import { isValidRawTx } from 'libs/validators';
+
+export function signRawTxWithPrivKey(privKey: Buffer, rawTx: Object): string {
```

Comment:
We should define a rawTx flow type, especially since there's a function that outlines what one should have.